### ISSUE TITLE
pixiv changed their page format

### ISSUE THREAD

**Issue by AshtonHarding** [sliceofcake/PixivMediaScraper#17]
> WARNING : could not find username on page - falling back to solely userID [developer's fault - pixiv changed their artist page format]

Putting this here. If I can find time, I'll try to fix it.

**Comment by sliceofcake** [comment]
Ugh they totally changed their UI...
Thanks for reporting. Will fix this soon. I haven't refreshed my folder for a few weeks, otherwise I would have normally caught this sooner.

**Comment by sliceofcake** [comment]
Looks like it wasn't just their UI. They're relying a lot AJAJ now, and the initially loaded page data has almost no information. They did a similar shift in December 2016 with asynchronous image loading, but this is way worse.

The nice thing is that they're using their own API now. Will see what the rules of ~this~ game are regarding limits/throttling. At least the API links weren't too bad to find and the data returned seems reasonable at first glance.

**Comment by sliceofcake** [comment]
The new format is now supported. It will work whether your account is on the old or the new format.

I tested it successfully in a few handpicked cases and I also tested it successfully with my personal folder. Please try it yourself and let me know if everything's working for you as well.

**Comment by AshtonHarding** [comment]
Oh thank you, for some reason I was not getting told I got responded to. I'm sorry about being 20 days later. I was actually coming over to post that I was able to use `wget` when I added:
`--referer='https://www.pixiv.net/member_illust.php?mode=manga_big&illust_id=ID_HERE&page=0'` to it.

I'm running the script right now and it appears to be working. I have a pretty big backlog here lol, but it's working just fine:
```
541 directories, 166943 files
541 directories, 167548 files
```
